ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-main.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: search-operator-bundle
+promotion:
+  name: "2.7"
+  namespace: stolostron
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-operator-bundle
+    env:
+      IMAGE_REPO: search-operator-bundle
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
```

Comment:
Can you add the e2e test here? For now it will only call our dummy e2e make target.